master/usage/openstack/labels.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Endpoint Labels
+canonical_url: 'https://docs.projectcalico.org/v3.2/usage/openstack/labels'
```

Comment:
@emanic Have I got this right?

If I have, is there anything I can do to avoid htmlproofer errors before the v3.2 pages are cut?

Reply:
I think it would probably be better to set the canonical URL to the master copy, rather than the v3.2 copy. We do have a sed command that the person doing the release runs, and this can be run twice, once to replace master with v3.2 and another time to replace v3.1 with v3.2. cc @tomdee as I think he will be managing this release. If you set this canonical URL to master, it should pass htmlproofer, as we've added a flag that resolves the chicken egg issues.